src/devices/schneider_electric.ts

```diff
@@ -2030,6 +2032,53 @@ const definitions: DefinitionWithExtend[] = [
             schneiderElectricExtend.visaKeyEventNotification('4'),
         ],
     },
+    {
+        zigbeeModel: ['NHMOTION/UNIDIM/1'],
```

Comment:
Please notice that this is different on the Swedish model.

Reply:
You have the newer multiwire version (`NHMOTION/DIMMER/1`). It's a separate device that has basically identical functionality except for improved dimming range. I'm working on a PR for it.

Reply:
Aah ok, please notify me then. Would be super if we could co-op on the implementation.

Reply:
This I'm missing from this PR though:

1. `dimmer_mode`: Auto, RL-LED
2. `on_level` (I would say "Change On Mode": 100%, Last level
3. Sensitivity: High, Medium,  Low
4. Detection delay: X seconds or minutes

Reply:
@enoch85 `dimmer_mode` works for my Elko dimmer. `on_level`, sensitivity and detection delay I have not figured out how to support.

Reply:
> @enoch85 `dimmer_mode` works for my Elko dimmer. `on_level`, sensitivity and detection delay I have not figured out how to support.

Yes, it's working, but I'm missing 'Auto', and I don't see RC in my Wiser app.